stan/math/prim/fun/mdivide_right_spd.hpp

```diff
@@ -33,9 +33,12 @@ inline Eigen::Matrix<return_type_t<T1, T2>, R1, C2> mdivide_right_spd(
     const Eigen::Matrix<T1, R1, C1> &b, const Eigen::Matrix<T2, R2, C2> &A) {
   static const char *function = "mdivide_right_spd";
   check_multiplicable(function, "b", b, "A", A);
-  check_positive(function, "rows", A.rows());
   check_symmetric(function, "A", A);
   check_not_nan(function, "A", A);
+  if (A.size() == 0) {
+    return Eigen::Matrix<return_type_t<T1, T2>, R1, C2>(b.rows(), 0);
+  }
+
   // FIXME: After allowing for general MatrixBase in mdivide_left_spd,
```

Comment:
[optional---this PR didn't introduce the problem]
These should say TODO and have a developer name attached, as in `TODO(mcol)`.  At least that's the Google coding standard we've adopted.

Reply:
That goes back to 2015 or so.